posthog/api/user.py

```diff
@@ -116,6 +116,11 @@ def validate_password_change(
 
         return password
 
+    def validate_is_staff(self, value: bool) -> bool:
+        if not self.context["request"].user.is_staff:
+            raise PermissionDenied("You are not a staff user, contact your instance admin.")
+        return value
```

Comment:
Is this used somewhere?

Reply:
I believe so, if you try to update yourself using `/@me` to staff.